Updated: App backend directions and connections

AnnanLao opened the pull request:
1. All 8 directions button are declared so that they are named the same as in xml file.
2. Any javaDoc is not committed. Those generated html files are not included.
3. Please review this commit and disregard App-backend branch.

AnnanLao committed:
Updated: App backend directions and connections

1. All 8 directions button are declared so that they are named the same as in xml file.
2. Any javaDoc is not committed. Those generated html files are not included.
3. Please review this commit and disregard App-backend branch.

livmari commented:
The main java file is missing comments. Please add them to structure the code for readability :)

livmari commented:
The connection method is has ID naming inconsistencies on line 37. The java file needs to handle connection to headset and car independent of each other.

AnnanLao committed:
Update main activity

Added comment to explain getBytes method  and other comment
The bluetooth connection specifies car.
